import_specifications/schema/dts_manifest.json

```diff
@@ -0,0 +1,39 @@
+{
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
+    "type": "object",
+    "properties": {
+        "resources": {"type": "array"},
+        "instructions": {
+            "type": "object",
+            "properties": {
+                "protocol": {"type": "string"},
+                "objects": {
+                    "type": "array",
+                    "items": {
+                        "type": "object",
+                        "properties": {
+                            "data_type": {"type": "string"},
```

Comment:
CV?

Reply:
Not here. This is setting what data type to treat the object as. Following the rules for xsv importers, that doesn't get set as some pre-determined value.

Reply:
:+1: